packages/wonder-blocks-pill/src/components/pill.tsx

```diff
@@ -147,57 +155,89 @@ const pillStyles = StyleSheet.create({
         width: "fit-content",
     },
     wrapperSmall: {
-        paddingLeft: Spacing.xSmall_8,
-        paddingRight: Spacing.xSmall_8,
-        borderRadius: Spacing.xxSmall_6,
+        paddingLeft: tokens.spacing.xSmall_8,
+        paddingRight: tokens.spacing.xSmall_8,
+        borderRadius: tokens.spacing.xxSmall_6,
+        // Minimum tap area recommendation for a11y
+        height: tokens.spacing.large_24,
```

Comment:
**question:** I remember that one of the changes was to make this `20px` tall. So if I understand correctly, we are using 24 instead for the reasons you stated in the comment right? If that's the case, it would be worth checking how that looks in the places where we are hard-coding 20px.

Reply:
I believe that was originally the case, but then we were asked to just bump it up to 24 in this thread:
https://khanacademy.slack.com/archives/C8Z9DSKC7/p1693925664214919?thread_ts=1693923761.646739&cid=C8Z9DSKC7

I can go ahead and use a snapshot release to check that it still looks okay in webapp.

Reply:
Got it, that makes sense, thanks!